docs/src/Modal.doc.js

```diff
@@ -23,85 +32,497 @@ card(
+          sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+        ></iframe>
+        `}
+      />
+
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="Limit the number of actions in a Modal. A primary and secondary action should be used for Modals. The rarely used tertiary actions are often destructive, such as “Delete”."
+        defaultCode={`
+          <iframe src="https://codesandbox.io/embed/cool-hill-ngdjx?fontsize=14&hidenavigation=1&theme=light&view=preview"
+          style={{width: '100%', height:'500px', border:'0', borderRadius: '4px', overflow:'hidden'}}
+          title="Modal with actions"
+          allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+          sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+        ></iframe>
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="In the few cases where Modals are being used within the Pinner product, aim to prevent the content from needing to scroll at a reasonable screen size."
+        defaultCode={`
+          <iframe src="https://codesandbox.io/embed/fervent-rosalind-xrzm5?fontsize=14&hidenavigation=1&theme=light&view=preview"
+          style={{width: '100%', height:'500px', border:'0', borderRadius: '4px', overflow:'hidden'}}
+          title="Modal without scrolling"
+          allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+          sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+        ></iframe>
+        `}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description="Use Modal for content that should have a dedicated surface, like log in flows. Think about the core areas of your product that could appear in navigation. If a dedicated URL would be beneficial, use a full page instead. If the user interaction is an optional sub-task, consider using a [Sheet](/Sheet)."
+        defaultCode={`
+
+`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description="Use Modal for long and complex tasks. Don’t keep the user in a Modal that takes multiple steps to exit. If multiple tasks are required, take the user to a separate page instead."
+        defaultCode={`
+
+`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description="Avoid adding additional Modals to the product. While these are currently used in some Pinner surfaces for editing, consider using a full page, Sheet, Flyout or inline editing for a better user experience."
```

Comment:
Given our usual format, this reads as "Don't avoid adding additional modals…", which I think is the opposite of our intention. Also: is this specifically talking about Pinner product? Otherwise our style guide recommends against being that specific.

Reply:
I think for Sheet and Modal we've had to get a bit more specific since we have specific guidelines depending on the surface. We mention it again for sizing and alignment on both as well

Reply:
Specificity is fine, just making sure it's intended 🙂